com.matthiasn.lotti.yml

```diff
@@ -0,0 +1,201 @@
+  sources:
+  - type: archive
+    url: https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.20.5/libsecret-0.20.5.tar.gz
+    sha256: b33b9542222ea8866f6ff2d31c0ad373877c2277db546ca00cc7fdda9cbab1c3
+- name: libkeybinder
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+    sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+- name: noto-emoji-fonts
+  buildsystem: simple
+  build-commands:
+  - mkdir -p /app/share/fonts/truetype
+  - mkdir -p /app/etc/fonts/conf.d
+  - install -Dm644 NotoColorEmoji.ttf /app/share/fonts/truetype/NotoColorEmoji.ttf
+  - install -Dm644 75-noto-color-emoji.conf /app/etc/fonts/conf.d/75-noto-color-emoji.conf
+  - fc-cache -fv /app/share/fonts
+  sources:
+  - type: file
+    url: https://github.com/googlefonts/noto-emoji/raw/v2.051/fonts/NotoColorEmoji.ttf
+    sha256: 72a635cb3d2f3524c51620cdde406b217204e8a6a06c6a096ff8ed4b5fd6e27b
+  - type: file
+    path: 75-noto-color-emoji.conf
+- rustup-1.83.0.json
+- name: lotti
+  buildsystem: simple
+  build-options:
+    append-path: /var/lib/flutter/bin:/var/lib/rustup/bin:/usr/lib/sdk/rust-stable/bin:/run/build/lotti/.cargo/bin:/usr/lib/sdk/llvm20/bin
+    prepend-ld-library-path: /usr/lib/sdk/llvm20/lib
+    env:
+      PUB_CACHE: /run/build/lotti/.pub-cache
+      FLUTTER_ROOT: /run/build/lotti/flutter_sdk
+      PATH: /var/lib/rustup/bin:/usr/lib/sdk/rust-stable/bin:/var/lib/flutter/bin:/run/build/lotti/flutter_sdk/bin:/run/build/lotti/.cargo/bin:/usr/lib/sdk/llvm20/bin:/usr/bin:/bin
+      CARGO_HOME: /run/build/lotti/.cargo
+      RUSTUP_HOME: /var/lib/rustup
+      PKG_CONFIG_PATH: /app/lib/pkgconfig:/usr/lib/pkgconfig
+      FLUTTER_SUPPRESS_ANALYTICS: 'true'
+  build-commands:
+  - if [ -d /var/lib/flutter ]; then cp -r /var/lib/flutter /run/build/lotti/flutter_sdk;
+    fi
+  - setup-flutter.sh -C /var/lib
+  - chmod -R u+w /run/build/lotti/flutter_sdk
+  - /run/build/lotti/flutter_sdk/bin/flutter pub get --offline
+  - mkdir -p .cargo
+  - ln -sfn ../cargo .cargo/cargo
+  - cp cargo/config .cargo/config.toml 2>/dev/null || true
+  - export CMAKE_PREFIX_PATH=/app:$CMAKE_PREFIX_PATH
+  - /run/build/lotti/flutter_sdk/bin/flutter build linux --no-pub --release --verbose
+  - "arch=\"x64\"\nif [ \"${FLATPAK_ARCH}\" = \"aarch64\" ]; then\n  arch=\"arm64\"\
+    \nfi\ncp -r build/linux/${arch}/release/bundle/* /app/\n"
+  - find /app -name "*1024*" -type f -delete
+  - chmod +x /app/lotti
+  - install -D linux/com.matthiasn.lotti.desktop /app/share/applications/com.matthiasn.lotti.desktop
+  - install -D flatpak/com.matthiasn.lotti.metainfo.xml /app/share/metainfo/com.matthiasn.lotti.metainfo.xml
+  - sed -i "s/{{LOTTI_VERSION}}/0.9.736/g" /app/share/metainfo/com.matthiasn.lotti.metainfo.xml
+  - sed -i "s/{{LOTTI_RELEASE_DATE}}/2025-11-20/g" /app/share/metainfo/com.matthiasn.lotti.metainfo.xml
```

Comment:
as you ignored it when I said it before: this should be set upstream prior to the actual release.

Also don't use `sed`.